lib/models/device.coffee

```diff
@@ -101,13 +101,26 @@ getDeviceModel = (deps, opts) ->
 
 	# Internal method for uuid/id disambiguation
 	# Note that this throws an exception for missing uuids, but not missing ids
+	# method for staged-releases get-device-id.sh
```

Comment:
I think this comment is probably just your own notes, right? I think this can go.